xarray/tutorial.py

```diff
@@ -18,9 +20,17 @@
 _default_cache_dir = _os.sep.join(('~', '.xarray_tutorial_data'))
 
 
+def _md5(fname):
```

Comment:
I wouldn't stress about that. Given that we use it in the script in xarray-data, it's fine either way for me.